src/imgix-core-js.js

```diff
@@ -176,11 +176,10 @@
     ImgixClient.prototype._buildDPRSrcSet = function(path, params) {
         var srcset = '';
         var targetRatios = [1, 2, 3, 4, 5];
-        var url = this.buildURL(path, params);
         
         for(var i = 0; i < targetRatios.length; i++) {
-          currentRatio = targetRatios[i];
-          srcset += url + ' ' + currentRatio +'x,\n'
+          params.dpr = i + 1;
```

Comment:
Feels weird to use `i + 1` here but `targetRatios[i]` below, when we're actually trying to make the two match up.